index.html

```diff
@@ -3554,24 +3554,30 @@ <h5>JWT Encoding</h5>
 in the <a>verifiable credential</a> <a>subject</a>.
               </li>
               <li>
-<code>aud</code> MUST represent the <code>subject</code> <a>property</a> of the
-consumer of the <a>verifiable presentation</a>.
+<code>aud</code> MUST represent (i.e., identify) the intended audience
+of the <a>verifiable presentation</a> (i.e., the Verifier(s) intended by
+the Presenter to receive and verify the <a>verifiable presentation</a>).
               </li>
             </ul>
 
             <p>
-Other JOSE header parameters and <a>claim</a> names not specified herein can be
-used if their use is not explicitly discouraged. Additional <a>claims</a> MUST
-be added to the <code>credentialSubject</code> <a>property</a> of the JWT.
-            </p>
+Other JOSE header parameters and JWT <a>claim</a> names not specified herein can be
+used if their use is not explicitly discouraged. Implementers MUST add claims about
+the <a>subject</a> to the <code>credentialSubject</code> property of the JWT
+<code>vc</code> <a>claim</a>, and <a>properties</a> of the
+<a>verifiable credential<a> to the <code>vc</code> <a>claim</a>. Such <a>claims</a>
+about the <a>subject</a> and <a>properties</a> of the <a>verifiable credential<a>
+MAY be echoed/repeated, but should not be placed solely in the JOSE header parameters
```

Comment:
This is additional normative guidance, and I'm concerned that it'll lead to non-interoperable JWT implementations. It's basically saying "some implementations may echo/repeat JOSE header parameters or JWT claims... but others may not echo/repeat, or may not even echo/repeat the same things." That will inevitably lead to some implementations doing the opposite of what other implementations do. I suggest we don't change this as it doesn't clarify things and only further confuses the issue.